### PR TITLE
ITM-484

### PR DESCRIPTION
**Pull request by Ayal-Ciobotaru**
Adjusted injuries and injury locations.

Please also review the client PR.

### CODE REVIEW

File: swagger_server/itm/itm_action_handler.py [hunk 1]
```diff
@@ -52,16 +52,18 @@ def _reveal_injuries(self, source: Character, target: Character):
             target.unstructured = source.unstructured_postassess
 
     def __successful_treatment(self, treatment: str, injury_name: str, location: str) -> bool:
-        # NOTE: Asthmatic, Forehead Scrape, Ear Bleed, an Internal injuries are currently untreatable.
+        # NOTE: Asthmatic, Forehead Scrape, Ear Bleed, Traumatic Brain Injury, Open Abdominal Wound and Internal injuries are currently untreatable.
```
Comment (nextcen-dgemoets) [inline swagger_server/itm/itm_action_handler.py:55]: Remove "Forehead Scrape," and add an Oxford comma after "Open Abdominal Wound."
Change "Forehead Scrape (Abrasion)" below to "Face Laceration".
Apart from the comma, this is all part of "remove Abrasion," but there's no way you would have known that.

File: swagger_server/itm/itm_action_handler.py [hunk 2]
```diff
@@ -52,16 +52,18 @@ def _reveal_injuries(self, source: Character, target: Character):
...
         """
             Head Injuries
             Forehead Scrape (Abrasion): Pressure bandage
             Face Shrapnel: Nasopharyngeal airway
             Ear Bleed: None
+            Traumatic Brain Injury: None
 
             Neck Injuries
             Neck Puncture: Hemostatic gauze
+            Neck Burn: Burn Dressing
```
Comment (nextcen-dgemoets) [inline swagger_server/itm/itm_action_handler.py:66]: I hadn't added Burn locations to these comments, but it's a good idea.  Please add for:
- Arm Injuries -> Forearm Burn
- Leg Injuries -> Calf Burn
- Leg Injuries -> Thigh Burn
- Chest Injuries -> Chest Burn

Also, please remove "Broken Forearm" and "Broken Wrist".

**Review by nextcen-dgemoets — COMMENTED** [review]
Looks good-- just some minor things that you couldn't have known of.

**Review by dereknop — APPROVED** [review]
lgtm

**Review by nextcen-dgemoets — APPROVED** [review]

**Pull request merged**